File: data/imported/knowledge/cells/java/merged/snippets/snippet_071.java
@Test
    void CV_21_doubleFormatsWithDecimal() {
        Workbook wb = new Workbook();
        Cell cell = wb.getWorksheets().get(0).getCells().get("A1");
        cell.putValue(1.5);
        assertEquals("1.5", cell.getStringValue());
    }

File: data/imported/knowledge/cells/cpp/merged/snippets/snippet_071.cpp
TEST(WorkbookPortingTests, NewWorkbookHasDefaultWorksheetAndBlankCells)
{
    Workbook workbook;
    auto& sheets = workbook.GetWorksheets();

    ASSERT_EQ(1, sheets.GetCount());
    EXPECT_EQ("Sheet1", sheets[0].GetName());

    auto cell = sheets[0].GetCells()["Z99"];
    EXPECT_EQ(CellValueType::Blank, cell.GetType());
    EXPECT_TRUE(cell.GetValue().IsEmpty());
    EXPECT_EQ("", cell.GetStringValue());
}